docs/source/6_api_reference/api.rst

```diff
@@ -56,3 +56,7 @@ misc
 
 .. automodule:: shimmingtoolbox.cli.referencemaps
    :members:
+
+.. click:: shimmingtoolbox.cli.mask:mask_cli
```

Comment:
`mask_cli` does not exist, it is named `mask`.

Reply:
The name of my function in mask.py is now changed to mask_cli